### ISSUE TITLE
figma concept - ix Figma Library

### ISSUE THREAD

**Issue by sn0wcat** [amosproj/amos2024ss01-xcelerator-demo-app#26]
## User story

1. As a designer
2. I want / need to design the screens of the application  using Siemens iX templates in figma
3. So that product owners can make decisions together with the industry partner.

cc: @danielleroux @silviowolf can you help here? Could we give the student team a preview version of the open source version or the siemens one? 

## Acceptance criteria

- [ ] ix Figma Library was provided to the project

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release

**Comment by sn0wcat** [comment]
@shahraz1998 @lamara11 Let's track the question about figma here.

**Comment by IngoSternberg** [comment]
Isn't it just the open source library linked at the bottom [here](https://ix.siemens.io/docs/design-kit/)?
I downloaded it but as far as I know Figma Professional is required to use libraries properly

**Comment by sn0wcat** [comment]
@IngoSternberg those are sketch libraries not figma libraries :)

**Comment by silviowolf** [comment]
Hi @sn0wcat, @Sabo2k, @IngoSternberg, I am not sure how the working mode between Siemens and students is set up but I see 3 ways to provide the libraries:

**Provide Figma guest accounts with editor rights for the students**
- students would get direct access to iX libraries but would also be forced to work inside Siemens’ Figma
- to be clarified: booking of possible costs for the editor licenses

**Export iX libraries and import them to another (student’s/univesrity’s) Figma**
- not easy to handle, libraries have dependencies, this dependencies need to be re-linked after import
- library updates are hard to apply, need the same export-import procedure

**Provide iX libraries via Figma Community libraries**
- that’s the intended way we want to provide our library for the open source community
- to be clarified: legal issues, check if it works with linked libraries
- updates are easier than the import-export procedure but not as easy as working directly inside the Siemens Figma

We could also set up a meeting to discuss and clarify the options.
